src/allotropy/parsers/agilent_gen5_image/agilent_gen5_image_structure.py

```diff
@@ -250,11 +250,32 @@ def _get_read_type(cls, procedure_details: str) -> ReadType:
 
 
 def create_results(
-    result_lines: list[str],
+    result_lines: list[str] | None,
     header_data: HeaderData,
     read_data: ReadData,
     sample_identifiers: dict[str, str],
 ) -> list[MeasurementGroup]:
+    if not result_lines:
+        return [
+            MeasurementGroup(
+                measurement_time=header_data.datetime,
+                plate_well_count=0,
+                analytical_method_identifier=header_data.protocol_file_path,
+                experimental_data_identifier=header_data.experiment_file_path,
+                measurements=[
+                    _create_measurement(
+                        "N/A",
+                        header_data,
+                        read_section,
+                        instrument_settings,
+                        None,
```

Comment:
Let's set `sample identifier` to `METATADA ONLY` (@james-leinas to review wording)